### ISSUE TITLE
Is there any support for aws profiles

### ISSUE THREAD

**Issue by rverma-nikiai** [stevearc/dql#15]
Is there any support for aws profile like passing profile with -p.

**Comment by stevearc** [comment]
No, we're not passing any auth data to the boto session when it's created, so nothing on the command line will end up there. Boto will read from [a number of sources](http://boto3.readthedocs.io/en/latest/guide/configuration.html#configuring-credentials), so you also have the options of `~/.aws/config` or `~/.boto`, for example

If you're sick of manually setting the profile for dql you can pretty easily `alias dql='AWS_DEFAULT_PROFILE=myprofile dql'`

**Comment by jspreddy** [comment]
I run it like this and it works: `AWS_PROFILE=my-profile dql -r us-east-1`
